wordpress/wp-content/plugins/cds-wpml-mods/src/Api/Endpoints.php

```diff
@@ -191,4 +201,38 @@ public function getTranslation(WP_REST_Request $request): WP_REST_Response|WP_Er
 
         return rest_ensure_response($response);
     }
+
+    /**
+     * Get the existing translation for a given page.
+     * Returns a WP_Error if no post is found for the id given.
+     *
+     * @param  WP_REST_Request  $request
+     *
+     * @return array | WP_Error
+     */
+    public function unsetTranslation(WP_REST_Request $request)// : WP_REST_Response|WP_Error
+    {
+        $post = get_post($request['id']);
+
+        if (is_null($post)) {
+            return new WP_Error('post_not_found', __('The post you are looking for does not exist', 'cds-wp-mods'), array( 'status' => 404 ));
+        }
+
+        $postType = 'post_' . $post->post_type;
+        $language_code = $this->post->getLanguageCodeOfPostObject($post);
+
+        // Unset existing translation if exists
+        if ($translatedPostID = $this->post->getTranslatedPostID($post)) {
```

Comment:
Might want to wrap this is a try / catch --- to ensure the `200` is actually valid

Reply:
That works, added it to all of our calls.